rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -149,6 +149,30 @@ TensorInfo::TensorInfo(std::vector<size_t> dims,
     if (_num_of_dims <= 3) _is_image = false;
 }
 
+TensorInfo::TensorInfo(std::vector<size_t> dims,
+                       RocalMemType mem_type,
+                       RocalTensorDataType data_type,
+                       RocalTensorlayout layout,
+                       RocalColorFormat color_format)
+    : _type(Type::UNKNOWN),
+      _dims(dims),
+      _mem_type(mem_type),
+      _data_type(data_type),
+      _layout(layout),
+      _color_format(color_format) {
+    _batch_size = dims.at(0);
+    _num_of_dims = dims.size();
+    _strides.resize(_num_of_dims);
+    _strides[_num_of_dims - 1] = tensor_data_size(data_type);
+    for (int i = _num_of_dims - 2; i >= 0; i--) {
```

Comment:
why the stride is not computed for the last 2 dims?

Reply:
The stride excludes only 1 dimension, i.e batch size there